.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -36,24 +36,30 @@ assignees: ''
 - Starship version: [the output of `starship --version`]
 - Shell type: [fish, zsh]
 - Shell version: [the output of `fish --version` or `zsh --version`]
-- Shell plugin manager: [e.g. oh-my-fish, oh-my-zsh, fisher, antigen]
+- Shell plugin manager: [if present, e.g. oh-my-fish, oh-my-zsh, fisher, antigen]
 - Terminal emulator: [e.g. iTerm, Hyper, Terminator]
 - Operating system: [e.g. macOS 10.13.4, Windows 10]
 
 #### Relevant Shell Configuration
 <!-- 
   Based on the shell you use, please paste the appropriate configuration.
-    Fish users: ~/.config/fish/config.fish
-    Zsh users: ~/.zshrc
+  The default location for your shell is:
+    Bash: ~/.bashrc
+    Zsh: ~/.zshrc
+    Fish: ~/.config/fish/config.fish
+    Xonsh: ~/.config/xonsh/rc.xsh
+    Elvish: ~/.config/elvish/rc.elv
+    Nushell: ~/.config/nu/config.toml
+    Ion: ~/.config/ion/initrc
 -->
 
-```bash
+```
 # Your configuration here
 ```
 
 #### Starship Configuration
 <!-- Can be found in  ~/.config/starship.toml -->
 
-```bash
+```
```

Comment:
Looks like we can confidently add syntax highlighting here:

```suggestion
```toml
```

Reply:
Good point!